src/Stripe.net/Entities/Invoices/InvoiceLineItem.cs

```diff
@@ -70,5 +70,8 @@ public class InvoiceLineItem : StripeEntity<InvoiceLineItem>, IHasId, IHasMetada
 
         [JsonProperty("type")]
         public string Type { get; set; }
+
+        [JsonProperty("unified_proration")]
+        public bool UnifiedProration { get; set; }
```

Comment:
Can you add the documentation for that field? We're trying to at least be complete on new fields. While not perfect it helps a bit over time!

Reply:
added